charts/komodor-agent/helm-fan-out.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -e
+# helm-fan-out
```

Comment:
what is this used for? couldnt find anything searching by text

Reply:
Copy/paste from the old chart - removing this file